manifests/k8s.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todo-list
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: todo-list
+  template:
+    metadata:
+      labels:
+        app: todo-list
+    spec:
+      containers:
+        - image: okteto.dev/todo-list:1.0.0
```

Comment:
don't set the image, let `kustomize` do it for you

Reply:
was needed during okteto deploy --build